onnx/defs/math/defs.cc

```diff
@@ -396,7 +396,7 @@ OPERATOR_SCHEMA(Max)
 Element-wise max of each of the input tensors. All inputs and outputs must
 have the same shape and data type.
 )DOC")
-    .Input(0, "data_0", "First of the input tensors.", "T")
+    .Input(0, "data_0", "First of the input tensors.", "T", OpSchema::Variadic)
```

Comment:
we should change "First" here and in other places below

Reply:
Yep. FIxed and Thank you!